robber/matchers/called.py

```diff
@@ -0,0 +1,21 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class Called(Base):
+    """
+    expect(function).to.be.called()
+    """
+
+    def matches(self):
+        try:
+            return self.actual.called
+        except AttributeError:
+            raise TypeError('{function} is not a mock'.format(function=self.actual))
```

Comment:
Very small thing but `mock` is actually object not function, I prefer to use `actual` as the name here instead of function here.
```python
raise TypeError('{actual} is not a mock'.format(actual=self.actual))
```